pysph/sph/rigid_body.py

```diff
@@ -299,49 +300,271 @@ def loop(self, d_idx, d_m, d_rho, d_au, d_av, d_aw,  d_p,
         s_fy[s_idx] += -d_m[d_idx]*ay
         s_fz[s_idx] += -d_m[d_idx]*az
 
+
+class LiuFluidForce(Equation):
```

Comment:
Because I added additional force due to solid particle, which is not given in Akinci's paper.

Now I added additional documentation about that